WalletWasabi.Gui/Tabs/WalletManager/LoadWalletViewModel.cs

```diff
@@ -276,27 +267,37 @@ public override void OnCategorySelected()
 				}
 
 				SelectedWallet = Wallets.FirstOrDefault();
-				SetWalletStates();
+				TrySetWalletStates();
 			}
 		}
 
-		private void SetWalletStates()
+		private bool TrySetWalletStates()
```

Comment:
The returned value is never used. It could return `void`.

Reply:
FooList.TryRemove(bar);

If a TrySomething can return a useful bool, then it should, even if it's not used. Not returning is an antipattern. I'm fine with this antipattern when it's out of convenience, if the function is so complex, returning a bool would be a pain and that bool would be never used, but intentionally doing so, when the correct pattern is just a few lines away is problematic.